cookbook/custom-page-data-provider.rst

```diff
@@ -84,7 +84,7 @@ is implemented, which filters the pages for a specific author:
                 - '@sulu_page.reference_store.content'
                 - '%sulu_document_manager.show_drafts%'
                 - '%sulu_security.permissions%'
-                - '@=container.hasParameter('sulu_audience_targeting.enabled')'
+                - '@=container.hasParameter(''sulu_audience_targeting.enabled'')'
```

Comment:
Thx for your pull request, I think `''` looks very uncommon for escaping can we use here normal `\` or `"` to make this:

```suggestion
                - '@=container.hasParameter(\'sulu_audience_targeting.enabled\')'
```

Or is this in yaml also not possible?

Reply:
That is indeed possible, there are 3 ways to use single quotes, using double single quotes to escape, backslashing for escaping or using double quotes around the value.